common/partition/default-partitioner.go

```diff
@@ -123,7 +130,14 @@ func pickIsolationGroup(wfPartition defaultWorkflowPartitionConfig, available ty
 	sort.Slice(availableList, func(i int, j int) bool {
 		return availableList[i] > availableList[j]
 	})
-	return pickIsolationGroupFallback(availableList, wfPartition)
+	r.metrics.IncCounter(metrics.PartitionerUnPinnedTask)
+	fallback := pickIsolationGroupFallback(availableList, wfPartition)
+	r.log.Debug("isolation group falling back to an available zone:",
```

Comment:
remove ':'
and we should log `partitionConfig`,
and there is a tag for available isolation groups

Reply:
good point, thank you